tools/mo/openvino/tools/mo/front/output_cut.py

```diff
@@ -35,9 +36,15 @@ def find_and_replace_pattern(self, graph: Graph):
                     if fw_info_value:
                         fw_info = fw_info_value
                         break
+
+            for out_idx in in_node.out_nodes():
+                out_node = in_node.out_node(out_idx)
+                out_nodes_ids[out_idx] = out_node.id
```

Comment:
'NoneType' object does not support item assignment

Reply:
Changed to dict.